src/workflow/orientation_screen.c

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string.h>
+
+#include "attestation.h"
+#include "idle_workflow.h"
+#include "orientation_screen.h"
+#include "password.h"
+#include "platform_config.h"
+#include "unlock.h"
+#include "workflow.h"
+
+#include <hardfault.h>
+#include <hww.h>
+#include <platform_config.h>
+#include <screen.h>
+#include <sd.h>
+#include <ui/components/confirm.h>
+#include <ui/components/info_centered.h>
+#include <ui/components/orientation_arrows.h>
+#include <ui/components/show_logo.h>
+#include <ui/components/waiting.h>
+#include <ui/screen_stack.h>
+#include <ui/workflow_stack.h>
+#if PLATFORM_BITBOXBASE == 1
+#include <usart/usart.h>
+#elif PLATFORM_BITBOX02 == 1
+#include <usb/usb.h>
+#endif
```

Comment:
remove? maybe double check if the other includes are all needed

Reply:
Reorganized everything 3a6a731